deploy/test/test_cases/TEST_ID_22_helm_rbac_defaults_taken_successfully.sh

```diff
@@ -26,4 +26,4 @@ $cli_with_timeout "get RoleBinding secrets-provider-role-binding"
 $cli_with_timeout "get ConfigMap cert-config-map"
 
 # Validate that the Secrets Provider took the default image configurations if not supplied and was deployed successfully
-$cli_with_timeout "describe job secrets-provider | grep 'cyberark/secrets-provider-for-k8s:1.1.0'" | awk '{print $2}' && $cli_with_timeout "get job secrets-provider -o jsonpath={.status.succeeded}"
+$cli_with_timeout "describe job secrets-provider | grep 'cyberark/secrets-provider-for-k8s'" | awk '{print $2}' && $cli_with_timeout "get job secrets-provider -o jsonpath={.status.succeeded}"
```

Comment:
@izgeri there needs to be a version here because we need to validate that the SP container took the default which is defined in the `values.yaml`. So in our case now, we want to verify that 1.1.1 is the version deployed and not 1.1.0. We planned on making a function to fetch the values.yaml but never got around to it. 

Anywho, we don't want to leave it without a version because if 1.1.0 by accident was taken then the tests would pass

Reply:
if you don't have the bandwidth then I think it would be best to add back the 1.1.1 and open an issue to fetch the tag from value.yaml that way, the test still has meaning. WDYT?

Reply:
Filed #270 - I'll fix this now and also add a reference to this file in the CONTRIBUTING release notes now.